cli/bootnode/boot_node.go

```diff
@@ -41,11 +43,15 @@ var StartBootNodeCmd = &cobra.Command{
 				MaxBackups: cfg.LogFileBackups,
 			},
 		)
+
 		if err != nil {
 			log.Fatal(err)
 		}
 
 		logger := zap.L()
+		log.Printf("starting SSV node (version %s)", commons.GetBuildData())
```

Comment:
Is there any reason not to use `logger` here? Why do we have two similar logs?

Reply:
you are right. changed.